### PR TITLE
fix typo for named-grid-lines.html styles

### PR DESCRIPTION
**Pull request by RusinovAnton**

### CODE REVIEW

File: areas.html [hunk 1]
```diff
@@ -3,11 +3,23 @@
...
-                                 "e e e e";
-            width: 150px;
+            grid-template-columns: repeat(3, 1fr) repeat(2, 2fr);
+            grid-template-rows: repeat(5, 1fr);
+            grid-template-areas: "a b b . ."
+                                 ". b b . c"
+                                 "d b b . c"
+                                 "d . . . c"
+                                 "e e e e e";
+            color: #f00;
```
Comment (mrego) [inline areas.html]: I'd use a different color than `red` as people can get confused about this being a mistake or not.
I think letting it black would be fine.

File: areas.html [hunk 2]
```diff
@@ -3,11 +3,23 @@
...
+            font-family: sans-serif;
+            font-size: 70px;
+            font-weight: bold;
```
Comment (mrego) [inline areas.html]: I don't care if you change the font sizes, it doesn't seem very relevant though.

**Review by mrego — COMMENTED** [review]
Hi @RusinovAnton, why do you close the PR?

I think the changes are fine, just adding some small comments. Thanks!

**Comment by mrego** [comment]
BTW, it'd be nice to update the screenshot if we're finally changing the grid areas example. Please provide a screenshot too. Thanks.

**Comment by RusinovAnton** [comment]
@mrego Hi, thanks for paying attention to this. First it were just "fix typo" PR. But then I just went playing with grid and broke it, later didnt find any time to proceed on this

**Pull request closed without merging**